scripts/lib/libvirt/admin-config.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+
+import libvirt
+import argparse
+import libvirt_helpers
+from string import Template
+
```

Comment:
If we are using flake8, I would suggest to follow some 'second-line' rules:

* Put first (in alphabetically order) the Python libraries (`argparse` and `string`)
* Separated with a line, put the other imports (also sorted) non-python libs (`libvirt`, `libvirt_helpers`)

https://www.python.org/dev/peps/pep-0008/#id17